src/schema/re_frame.cljs

```diff
@@ -5,6 +5,16 @@
 (def ^:private ?cofx
   [:map])
 
+(def ^:private ?event-fx
+  [:maybe
+   [:map {:closed true}
+    [:db {:optional true} [:maybe map?]]
+    [:fx {:optional true}
+     [:maybe
+      [:vector {:optional true}
```

Comment:
Are you sure `optional` here has any effect? I thought it only had a meaning in describing map values.

Apparently to me, we can be a bit more strict without causing trouble/refactors in our prod code:

```clojure
(def ^:private ?event-fx
  [:maybe
   [:map {:closed true}
    [:db {:optional true} map?]
    [:fx {:optional true}
     [:vector [:vector :any]]]]])
```

Reply:
 You're right, couldn't find any instance where optional is used anywhere else but map keys. Will remove it 👍